inputs/input_dfm.yaml

```diff
@@ -69,6 +69,11 @@ gru_lstm:
   # shuffle when creating DataLoader
   dl_shuffle: True 
   random_seed: 123
+  # Early stopping:
+  # Number of epochs with no improvement after which training will be stopped and
+  #   the model with the least validation loss will be saved
+  # If False, 0 or negative, early stopping will be deactivated
+  early_stopping_patience: False
```

Comment:
It's a tiny thing, but maybe we could say that `early_stopping_patience` cannot be `True` at the moment (as far as I have seen), but needs to be a number. As by default is `False` people might think that they could make it `True` and we will use a "default" patience - which we could do in theory, but I think we don't. But I might have missed something.

Reply:
Good point, I removed `False`, see: https://github.com/Living-with-machines/DeezyMatch/pull/74/commits/5d2f16e7e5e3be8c2a39b2c247d645bbb920ced2

Reply:
Is this OK?

Reply:
perfect